src/index.js

```diff
@@ -10,42 +10,60 @@ import { useForm, useField } from 'vee-validate';
  */
 export const mapElementsInSchema = (schema, fn) => schema.map(row => row.map(el => fn(el)))
 
-export default function VeeValidatePlugin (baseReturns) {
-  // Take the parsed schema from SchemaForm setup returns
-  const { parsedSchema, formBinds } = baseReturns
+/**
+ * Maps the validation state to props
+ */
+function defaultMapProps(validation, el) {
```

Comment:
Should we remove the unused `el` param here?

Reply:
Yea, I forgot it 😅